app/main.py

```diff
@@ -1,8 +1,34 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for human in people:
+        new_person = Person(human["name"], human["age"])
+        if human.get("wife"):
+            new_person.wife = human.get("wife")
+            person_list.append(new_person)
+        elif human.get("husband"):
+            new_person.husband = human.get("husband")
+            person_list.append(new_person)
+        else:
+            person_list.append(new_person)
```

Comment:
You should make `person_list.append(new_person)` one time
```suggestion
        if human.get("wife"):
            new_person.wife = human.get("wife")
        elif human.get("husband"):
            new_person.husband = human.get("husband")
        person_list.append(new_person)
```